concepts/graph-explorer/graph-explorer-features.md

```diff
@@ -10,9 +10,17 @@ author: bettirosengugi
 
 [Graph Explorer](https://developer.microsoft.com/graph/graph-explorer/) is a developer tool that lets you conveniently make Microsoft Graph REST API requests and view corresponding responses. This article describes how to use some of the important features in Graph Explorer.
 
-## Consent to permissions
+[Graph Explorer Sample Teams](TODO) exists alongside the web application to mock Teams app permissions by showing resource-specific consent and is used to support "Teams application" mode in the web application. 
 
-The user or administrator must grant Graph Explorer the correct permissions via a consent process to access data in Microsoft Graph. Consent to permissions in Graph Explorer either via the **Modify permissions** tab or the **Select permissions** option in the settings gear next to your profile when you’re signed in. 
+## Use Graph Explorer as sample Teams application
```

Comment:
Did we decide on calling it a 'sample app'? I remember previous discussions with Darrel 'proxy' or just 'as a Teams application' makes more sense? Or is more "intuitive"

Reply:
The name needs to be reviewed with Laura - Ayu has it on her list of things to cover with her